astropy/io/votable/tree.py

```diff
@@ -3447,7 +3449,7 @@ def extra_attributes(self):
         A dictionary of string keys to string values containing any
         extra attributes of the RESOURCE_ element that are not defined
         in the specification.  (The specification explicitly allows
-        for extra attributes here, but nowhere else.)
+        for extra attributes here, but nowhere else.).
```

Comment:
Maybe better to split at start of `(` -- and maybe remove parentheses?

Reply:
✅ I added a summary line.